### DISCUSSION TITLE
`log_model` under a run but pass the existing models from the registry instead of creating new model ?

### DISCUSSION THREAD

**Discussion by AlamHasabie** [mlflow/mlflow#14179] [Q&A]
Hi,
I'm wondering whether MLFlow supports registering references to models in model registry as part of a run ? This might happen on pipeline-based systems, i.e. several models on a pipeline setting that is evaluated together under a single run. The run itself does not produce any new model, but it uses several models in the model registry, so one might want to log those models as part of the run.

I do think I can have a workaround by, say, using the model URI as parameter of the model, however it just seems better to log the referenced model so we can click on it and search it. 

Thanks in advance !

**Comment by zakariajaadi** [comment]
Hi Alam,

I think MLflow currently does not natively support registering references to models in the Model Registry as part of a run. However, If you want actual clickable references, a workaround would be to log a dummy MLflow model artifact that contains metadata pointing to the referenced models, this dummy model would serve as a "container" for metadata or references to the actual models used in your pipeline, and since it's logged and registered in the Model Registry, it will be clickable in the MLflow UI. 

Best regards,
Zakaria
